modules/@apostrophecms/schema/ui/apos/components/AposInputSelect.vue

```diff
@@ -37,13 +37,29 @@ export default {
       default: 'menu-down-icon'
     }
   },
+  data() {
+    return {
+      choices: []
+    };
+  },
+  mounted() {
+    if (!this.field.choices.find(choice => {
+      return !choice.value || choice.value === 'any';
+    })) {
+      this.choices.push({
+        label: 'Any',
+        value: 'any'
+      });
+    }
+    this.choices = this.choices.concat(this.field.choices);
+  },
   methods: {
     validate(value) {
       if (this.field.required && !value.length) {
         return 'required';
       }
 
-      if (!this.field.choices.includes(value)) {
```

Comment:
This was erroring on every value since it wasn't looking at the `value` property.